requirements.txt

```diff
@@ -0,0 +1,43 @@
+alembic==1.8.1 ; python_version >= "3.9" and python_version < "4.0"
```

Comment:
Just a question for this, how are you exporting this requirements file? Can we consider using [this buildpack](https://elements.heroku.com/buildpacks/moneymeets/python-poetry-buildpack) which natively supports poetry so that we can automate the deployment process?

Reply:
oh yes i was finding out how to use poetry but settled for the following temporary hack

```bash
poetry export --without-hashes --format=requirements.txt > requirements.txt
```

will look into that buildpack